HACKING.md

```diff
@@ -38,14 +38,34 @@ separated by an underscore. For example, `messages_gherkin-use-protobuf`.
 
 ### Local Testing / Development checks
 
-To run a copy of the CI locally simply run the command below. **NB: This will require docker being installed.**
+Cucumber runs with a highly coupled system of tests. Because cucumber is a mono-repo, testing these is best done
+in a docker container, due to needing a large amount of pre-requisites and often setting them up can be tricky.
+
+To start with, we need to ensure we have Docker installed. This is usually fairly easy, and a comprehensive set
+of setup instructions are available [HERE](https://docs.docker.com/install/) for most OS'
+
+Once docker is installed, we then need to grab a copy of the official cucumber container. This should only need
+to be done once and will likely take 5-10 minutes.
+
+```bash
+$ ./scripts/docker-run Dockerfile.cucumber-build # This requires at least 2.5Gig of space on your directory
```

Comment:
It's `./scripts/docker-run Dockerfile` - the `.cucumber-build` suffix is gone now.